app/src/lib/shells/win32.ts

```diff
@@ -136,8 +136,8 @@ async function findPowerShellCore(): Promise<string | null> {
 
   const first = powerShellCore[0]
   if (first.type === RegistryValueType.REG_SZ) {
-    if (await pathExists(path)) {
-      return path
+    if (await pathExists(first.data)) {
```

Comment:
Why you changed this one from `path` to `first.data`, I suspect that this is the reason build fails. @saschanaz